frontend/src/components/stages/payout_summary_view.ts

```diff
@@ -112,15 +116,28 @@ export class PayoutView extends MobxLitElement {
     item: DefaultPayoutItemResult,
     currency: PayoutCurrency
   ) {
-    return html`
-      ${this.renderBaseAmountEarned(item, currency)}
-    `;
+    return html` ${this.renderBaseAmountEarned(item, currency)} `;
   }
 
   private renderChipPayoutItemResult(
     item: ChipPayoutItemResult,
     currency: PayoutCurrency
   ) {
+    const getInitialTotal = () => {
+      let total = 0;
+      for (const result of item.chipResults) {
+        let quantity = 0;
+        if (result.chip.name.includes('red')) {
```

Comment:
can we add a TODO to generalize this later? the initial chips should really be based on the chip stage's configuration (i.e., ChipStageConfig has a ChipItem list that specifies the starting quantities), not custom variables